addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2090,6 +2091,16 @@ var SnippetsMenu = Widget.extend({
             }],
         }).open();
     },
+    /**
+     * @private
+     */
+    _onMouseDown: function () {
+        $('#wrapwrap').css('pointer-events', 'none');
+        $('body').on('mouseup', function () {
+            $('#wrapwrap').css('pointer-events', 'auto');
+            $('body').off('mouseup');
```

Comment:
This removes all the mouseup handler from everywhere, you only want to remove the one you set here. You can use the `one` function of jQuery instead of `on`.

Need to be smarter than that too: if you mouseup over an element which stopPropagation for whatever stupid reason... the whole editor is locked forever and the user loses everything. Even with crappy code, this needs to be more robust to avoid locking the editor. Set the property, then start a timer to remove it. On mouseup, stop the timer and remove it directly.

Also, instead of changing pointer-events directly, use a dedicated class. What if another code decides to tweak the pointer events property too? It would conflict with yours, while it won't with a dedicated class.